tests/store/test_store_backends.py

```diff
@@ -167,3 +170,29 @@ def test_FixedLengthTupleFilesystemStoreBackend(tmp_path_factory):
     my_file_AAA
     my_file_BBB
 """
+
+@mock_s3
+def test_FixedLengthTupleS3StoreBackend():
+    path = "dummy_str"
+    bucket = "leakybucket"
+
+    # create a bucket in Moto's mock AWS environment
+    conn = boto3.resource('s3', region_name='us-east-1')
+    conn.create_bucket(Bucket=bucket)
+
+    my_store = FixedLengthTupleS3StoreBackend(
```

Comment:
Can you add a test to show what the s3 bucket and keynames are actually generated?

Something like:
`assert conn.has_key_etc("s3://some-bucket/other-stuff/my_file_AAA.txt")`

That'll be helpful as documentation, too.